README.md

```diff
@@ -351,6 +372,38 @@ Any other options will be read and passed to the build functions. Usefull exampl
 
 # Troubleshoot
 
+## Error `Requires: python(abi) = 2.6`
+
+The whole point of building so many RPMs is because we try to get ride of python2.6 in all the low level packages. If you see such a message, that means that one of the package you are building requires a dependency that we do not provide. And if that's the case, you might want to look at version change. For example::
+
+```
+Getting requirements for yum-3.2.29-81.el6.py27.usc4.src
+ --> python-2.7.13-2.el6.x86_64
+ --> gettext-0.17-18.el6.x86_64
+ --> intltool-0.41.0-1.1.el6.noarch
+ --> python-nose-0.10.4-3.1.el6.py27.usc4.noarch
+ --> python-2.7.13-2.el6.x86_64
+ --> rpm-python-4.8.0-59.el6.x86_64
+ --> Already installed : rpm-4.8.0-59.el6.x86_64
+ --> python-iniparse-0.3.1-2.1.el6.py27.usc4.noarch
+ --> python-2.7.13-2.el6.x86_64
+ --> python-urlgrabber-3.9.1-11.el6.py27.usc4.noarch
+ --> yum-metadata-parser-1.1.2-16.el6.py27.usc4.x86_64
+ --> pygpgme-0.1-18.20090824bzr68.el6.py27.usc4.x86_64
+Error: Package: rpm-python-4.8.0-59.el6.x86_64 (base)
+           Requires: libpython2.6.so.1.0()(64bit)
+Error: Package: rpm-python-4.8.0-59.el6.x86_64 (base)
+           Requires: python(abi) = 2.6
+           Installing: python-2.7.13-2.el6.x86_64 (local-py2.7)
+               python(abi) = 2.7
+ You could try using --skip-broken to work around the problem
+ You could try running: rpm -Va --nofiles --nodigest
+ ```
+
+ This stack trace is due to `rpm-python-4.8.0-59` being pulled from the `base` repo, despite we provide `rpm-python-4.8.0-55`. This is because some packages (`yum` in that case) has a loose dependency (no specific version ) on `rpm-python`, so it takes the latest. The solution is to update our version of `rpm-python`
+
```

Comment:
Maybe add some comments that you should compare the version here that is taken from (base) with that which is in the /diracos_repo folder and if they don't match....